tests/nested_pandas/nestedframe/test_io.py

```diff
@@ -459,3 +459,17 @@ def test__get_storage_options():
     assert storage_opts is not None
     # S3 should NOT have the block_size override (only HTTP/HTTPS)
     assert storage_opts.get("block_size") != FSSPEC_BLOCK_SIZE
+
+
+def test_list_struct_partial_loading_error():
+    """Test that attempting to partially load a list-struct raises an error."""
+    # Load in the example file
+    with pytest.raises(ValueError):
```

Comment:
Let's check that it is "our" ValueError by checking the message (`pytest.raises(ValueError, match=...)`). Because `pa.ArrowInvalid` is also `ValueError`